.chloggen/enumsasints.yaml

```diff
@@ -0,0 +1,11 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
Isn't this still a breaking change?

Reply:
The protocol required (no longer) to support to unmarshal both, so not sure why this is a "breaking change". Can mark if you feel that way.

Reply:
Ok sounds good.